src/config/settings/local.py

```diff
@@ -22,13 +22,18 @@
     ),
     "DEFAULT_RENDERER_CLASSES": (
         "djangorestframework_camel_case.render.CamelCaseJSONRenderer",
+        "rest_framework.renderers.BrowsableAPIRenderer",
     ),
+    # these authentication classes are for local only
     "DEFAULT_AUTHENTICATION_CLASSES": (
         "rest_framework.authentication.SessionAuthentication",
         "rest_framework.authentication.TokenAuthentication",
     ),
     "DEFAULT_PERMISSION_CLASSES": ("rest_framework.permissions.IsAuthenticated",),
     "UNAUTHENTICATED_USER": None,
+    "DEFAULT_SCHEMA_CLASS": "drf_spectacular.openapi.AutoSchema",
```

Comment:
Should we add it to both places ? Or we can just make the second object extend from the first one, i.e. local can add on to the base REST_FRAMEWORK dict, the overriding of the variable can make it confusing.

Reply:
Ohhh yes, it would absolutely be better that way